ssz-rs/src/uint.rs

```diff
@@ -93,6 +92,7 @@ impl U256 {
     }
 }
 
+#[cfg(feature = "serde")]
```

Comment:
not sure what this has to do w/ `serde`? this `From` impl is just a convenience to make it easier to construct `U256` values

if this pulls in some `std` feature then I'd rather focus on handling that and not just batch into the `serde` feature

Reply:
Whoops, I added this accidentally during the rebase. Removed in https://github.com/ralexstokes/ssz-rs/pull/25/commits/1ff4b88163df1e0178af8da0fc85c4e27d55f1d1.